<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1">
       <widget class="QComboBox" name="button_right">
        <item>
         <property name="text">
          <string>a</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>b</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>c</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>d</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>e</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>button left:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="button_down">
        <item>
         <property name="text">
          <string>a</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>b</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>c</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>d</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>e</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="button_left">
        <item>
         <property name="text">
          <string>a</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>b</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>c</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>d</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>e</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>button down:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="apply">
        <property name="text">
         <string>apply</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>button up:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>button right:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="button_up">
        <item>
         <property name="text">
          <string>a</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>b</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>c</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>d</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>e</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>analogic button:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="button_analogic">
        <item>
         <property name="text">
          <string>a</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>b</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>c</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>d</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>e</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
